app/src/test/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentEffectHandlerTest.kt

```diff
@@ -142,20 +142,32 @@ class ScheduleAppointmentEffectHandlerTest {
   }
 
   @Test
-  fun `when schedule sppointment for patient from next is received then schedule appointment`() {
+  fun `when schedule appointment for patient from next effect is received, then schedule appointment`() {
```

Comment:
Can we add `and mark older appointment as visited` in the test name as well?

Reply:
Ah right, previously it was implicit. Now that we have made it explicit in the effect handler that makes sense. Although I am wondering if adding those side effects increase the names of the tests. For example, the flow itself is scheduling an appointment, and a side effect of it is marking older appointments, that can still be read from the tests themselves. So were essentially verifying the flow rather than everything that might happen in there. WDYT? 

I can rename the test if you think that makes more sense and is easier to understand. I am fine with either.

Reply:
Umm, I personally prefer making the test names explicit. As, I look at tests as a representation of the business logic as well.

Reply:
Tests themselves will still represent that. Anyway, I have updated the names and pushed the changes.